lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -196,6 +196,9 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     @Output()
     siteChange: EventEmitter<string> = new EventEmitter<string>();
 
+    @Output()
+    disableUploadButton: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
can you use a different name for the event emitter? The panel doesn't know anything about the upload button that is part of the content-node-selector

Reply:
Updated